oshmem/shmem/c/shmem_g.c

```diff
@@ -67,6 +89,6 @@ SHMEM_TYPE_G(_longlong, long long, shmem)
 SHMEM_TYPE_G(_float, float, shmem)
 SHMEM_TYPE_G(_double, double, shmem)
 SHMEM_TYPE_G(_longdouble, long double, shmem)
-SHMEM_TYPE_G(_int16, int16_t, shmemx)
-SHMEM_TYPE_G(_int32, int32_t, shmemx)
-SHMEM_TYPE_G(_int64, int64_t, shmemx)
+SHMEM_TYPE_GX(_int16, int16_t, shmemx)
```

Comment:
@yosefe keep in mind that spec1.3 tells nothing about ext API.
Seems to be consistent reduce extensions API should be changed too. Right? (see
https://github.com/open-mpi/ompi-release/pull/1073/commits/2e11e8ff784bef9f642dc51150e6efe92c6cb0a4#diff-475037a625962c3ff765b9247497f712R241)